standard/docs/en/implementation/publication_policy.md

```diff
@@ -1,17 +1,17 @@
 # Publication policies
 
-Publishing Open Contracting data involves making a number of choices about what data and documents to include and how to classify and categorize different items. To allow users to understand the choices that a publisher has made, a publisher should provide a public web page or document that details the decisions they have made concerning the issues listed below.
+Publishing Open Contracting data involves making a number of choices about what data and documents to include and how to classify and categorize different items. To allow users to understand the choices that a publisher has made, a publisher ought to provide a public web page or document that details the decisions they have made concerning the issues listed below.
 
-This document should also contain information about future plans for changes to the data and who to contact with enquiries about the data.
+This document ought to also contain information about future plans for changes to the data and who to contact with enquiries about the data.
 
-Publication policy pages should include information about:
+Publication policy pages ought to include information about:
 
 * Who is responsible for providing this data - including appropriate contact details and locations of where to offer feedback.
 * How the data is generated, and how often - for example, a brief summary describing the systems the data is generated from, how often the source data is updated, and any processes that operate between the source data and the published open contracting data and documents.
-* Any exclusions from the data - such as information which is not disclosed for reasons of privacy and security. This may describe the fields which are excluded for a selection of records, and/or the policy applied to decide if any information should be withheld from a public OCDS release. 
+* Any exclusions from the data - such as information which is not disclosed for reasons of privacy and security. This can describe the fields which are excluded for a selection of records, and/or the policy applied to decide if any information is withheld from a public OCDS release. 
 * Any custom codelists used - or custom codes added to existing codelists. 
 * Future development plans - brief notes on the future plans of the publisher to improve their data.
 
-Publishers may also wish to apply for, and include the results of, an [Open Data Certificate](https://certificates.theodi.org/), on their publication policy pages.
+Publishers might also wish to apply for, and include the results of, an [Open Data Certificate](https://certificates.theodi.org/), on their publication policy pages.
 
 A link to the publication policy can be embedded within the release package or record package. 
```

Comment:
> A link to the publication policy **ought to** be embedded within the release package or record package. 

Since all fields in the schema are "should"

Reply:
Good catch. Fixed in 61f5cd6